packages/datetime/src/_datepicker.scss

```diff
@@ -112,12 +112,13 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
   .DayPicker-WeekNumber {
     display: table-cell;
     cursor: pointer;
-    min-width: 1rem;
-    padding: 0.5rem;
+    width: $datepicker-day-size;
+    height: $datepicker-day-size;
+    padding-right: 10px;
```

Comment:
General rule: use `$pt-grid-size` instead of `10px`.

Reply:
Changed, thanks!